tests/government-frontend.spec.js

```diff
@@ -19,4 +19,13 @@ test.describe("Government Frontend", { tag: ["@app-government-frontend"] }, () =
     await page.getByRole("button", { name: "Get emails about this page" }).first().click();
     await expect(page.getByText("You need a GOV.UK One Login to get these emails.")).toBeVisible();
   });
+
+  test("check application CSS loads", async ({ page, request }) => {
+    await page.goto("/government/consultations/soft-drinks-industry-levy");
```

Comment:
I think this is a consultation page? They're currently being migrated to frontend: https://github.com/alphagov/frontend/pull/5015

Unfortunately there's not a lot of other options to choose from, although there are some we've not started. I'll send you our progress tracker, see if there's a better one on there.

Reply:
Ahh, I see, I copied this page from an existing test which is set to be removed in the near future it looks like - https://github.com/alphagov/govuk-e2e-tests/pull/313/files

I'll update the test to check the `topical_event_about_page` in government-frontend instead as it looks like that is not being worked on right now